### PR TITLE
feat(sync-actions): add enableDiscounted option to product

### PR DESCRIPTION
**Pull request by markus-azer**
#### Summary

Add enableDiscounted option to product

#### Description

This PR is resolving this issue [1617](https://github.com/commercetools/nodejs/issues/1617)

### CODE REVIEW

File: packages/sync-actions/test/product-sync-prices.spec.js [hunk 1]
```diff
@@ -147,6 +147,61 @@ describe('Actions', () => {
       ])
     })
 
+    test('should build actions for prices with discounted when enableDiscounted is set to true', () => {
```
Comment (katmatt) [inline packages/sync-actions/test/product-sync-prices.spec.js:150]: Nice test 👍

**Review by katmatt — APPROVED** [review]
Great job and looks very good to me 👍

**Review by jenschude — APPROVED** [review]

**Review by daern91 — APPROVED** [review]
LGTM! And seems like the unit tests got broken in this PR: https://github.com/commercetools/nodejs/pull/1618

Should we fix that before merging this PR, maybe?

cc @ashishhk

**Comment by ashishhk** [comment]
> LGTM! And seems like the unit tests got broken in this PR: #1618
> 
> Should we fix that before merging this PR, maybe?
> 
> cc @ashishhk

Sure @daern91  unless @jenschude is already on it.

**Pull request merged**